File: Backend/Learnz/DTOs/TogetherUserProfileDTO.cs
namespace Learnz.DTOs;

public class TogetherUserProfileDTO
{
    public Guid UserId { get; set; }
    public string Username { get; set; }
    public Grade Grade { get; set; }
    public Guid ProfileImage { get; set; }
    public string Information { get; set; }
    public Subject GoodSubject1 { get; set; }
    public Subject GoodSubject2 { get; set; }
    public Subject GoodSubject3 { get; set; }
    public Subject BadSubject1 { get; set; }
    public Subject BadSubject2 { get; set; }
    public Subject BadSubject3 { get; set; }
}

File: Frontend/src/app/DTOs/Together/TogetherUserProfileDTO.ts
import { Grade } from "src/app/Enums/Grade";
import { Subject } from "src/app/Enums/Subject";

export interface TogetherUserProfileDTO {
  userId: string;
  username: string;
  grade: Grade;
  profileImagePath: string;
  information: string;
  goodSubject1: Subject;
  goodSubject2: Subject;
  goodSubject3: Subject;
  badSubject1: Subject;
  badSubject2: Subject;
  badSubject3: Subject;
}